Add minimum width for central Muon & FDA widgets

sf1919 opened the pull request:
**Description of work.**
The tab widget for Muon Analysis and Frequency Doman Analysis GUIs have been resized to allow the labels for each tab to be fully visible. This was raised by Instrument Scientists at a Muon Project Delivery Board meeting.

**Report to:** Muon Project Delivery Board

**To test:**
1. Open Muon Analysis
2. Check that the label for each tab is readable (see screenshot below as an example)

![Screenshot (10)](https://user-images.githubusercontent.com/55837273/127004787-a6ec1620-e2d7-4632-bee7-539d5f0e6bf2.png)


3. Open Frequency Domain Analysis
4. Check that the label for each tab is readable

*There is no associated issue.*

*This does not require release notes* because **it is a minor resizing of the GUI**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

sf1919 committed:
Add minimum width for central Muon & FDA widgets

Set the minimum width of the widget containing the tabs to make labels readable

tolu28-coder reviewed (dismissed):
Everything works as expected

thomashampson commented on scripts/Muon/GUI/FrequencyDomainAnalysis/frequency_domain_analysis.py:
```diff
@@ -134,6 +134,7 @@ def __init__(self, parent=None, window_flags=None):
         self.help_widget = HelpWidget(self.context.window_title)
 
         central_widget = QtWidgets.QWidget()
+        central_widget.setMinimumWidth(590)
```
Can you try putting this in the `setup_tabs()` function instead?
```
        self.tabs.setElideMode(QtCore.Qt.ElideNone)
        self.tabs.setUsesScrollButtons(False)
```
If it works, that would avoid any problems in the future if someone renamed a tab, for example.

sf1919 replied:
This works brilliantly and I agree - a far better approach as it will work with any future changes. I hadn't been happy with a fixed width but wasn't sure how else to resolve the problem. Many thanks.

thomashampson commented:
I think you should add this to the release notes since it visibly affects the GUI, and was requested by users.

sf1919 committed:
32075 changes made after review

A better way of dealing with tab labels being obscured was suggested during review and successfully implemented. Documentation also added.
